gui/utils/compat.py

```diff
@@ -1,262 +0,0 @@
-# Backport of OrderedDict() class that runs on Python 2.4, 2.5, 2.6, 2.7 and pypy.
```

Comment:
I mean, pyfa doesn't run correctly in 2.6 anyway. 2.7 is nearly 7 years old now. Go for it.